### ISSUE TITLE
Draft not preserved across screen orientation change

### ISSUE THREAD

**Issue by SilverPuppy** [FossifyOrg/Messages#114]
### Checklist

- [X] I can reproduce the bug with the latest version given [here](https://github.com/FossifyOrg/Messages/releases/latest).
- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Messages/issues) or [closed](https://github.com/FossifyOrg/Messages/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Messages/discussions) or [closed](https://github.com/FossifyOrg/Messages/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Messages/blob/master/CONTRIBUTING.md).

### Affected app version

1.0.1

### Affected Android/Custom ROM version

A12/Moto G stock

### Affected device model

Moto G Pure

### How did you install the app?

F-Droid / IzzyOnDroid

### Steps to reproduce the bug

1. Select a photo to send.
2. Compose a message but don't send it.
3. Change screen orientation. 

### Expected behavior

Draft message should be preserved, and appear in the new orientation. 

### Actual behavior

Entire draft message will disappear. This happened to me twice in a row this morning. The first time, I stopped composing to pet the cat, then looked back, and my message in progress was mysteriously gone. I started again, moved such that the screen orientation changed, and again, my work disappeared. This time I saw it happen. I tested the steps above to confirm the behavior, and it's consistent.

**Comment by SilverPuppy** [comment]
Weird, an orientation change just now with an empty message box retrieved an MMS draft from an hour ago, which had already been sent.

**Comment by SilverPuppy** [comment]
I've found MMS drafts accidentally by using the back button to navigate backward through the activity stack, something I rarely do, but did once absentmindedly and found very revealing. I wonder if it's a stack navigation mistake.

**Comment by Helsinki-Vice** [comment]
Not able to reproduce bug on this device:
* Android 14, OneUI 6.1 on Samsung A14 5G
* Messages version 1.1.5 via F-Droid
Observed behavior is as follows:
* Orientation changes destroys attachments, but not message content.
* Messages with attachments are not saved to drafts when the back button is pressed or the app is closed.

**Comment by naveensingh** [comment]
Yes, text content is now preserved after https://github.com/FossifyOrg/Messages/issues/13 but MMS attachments are still discarded.
